vagrant/build.sh

```diff
@@ -62,24 +70,17 @@ else
 fi
 
 # Create maestro dummy config if config does not exist
-[ -f maestro.config ] || \
-echo "network:
-    interfaces:
-        - if_name: eth1
-          exists: replace
-          dhcpv4: true
-          hw_addr: \"{{ARCH_ETHERNET_MAC}}\"
-        - if_name: eth2
-          exists: replace
-          dhcpv4: false
-          ipv4_addr: 10.0.102.102
-          ipv4_mask: 24
-          hw_addr: \"{{ARCH_ETHERNET_MAC}}\"
-devicedb_conn_config:
-    devicedb_uri: \"https://$DEVICE_ID:9090\"
-    devicedb_prefix: \"vagrant\"
-    devicedb_bucket: \"lww\"
-    relay_id: \"$DEVICE_ID\"
-    ca_chain: \"$DEVICEDB_SRC/hack/certs/myCA.pem\"
-config_end: true
-" >> maestro.config
+if [ ! -f maestro.config ]; then
+    SYMPHONY_HOST="gateways.mbedcloudstaging.net"
+    SYMPHONY_CLIENT_CRT=$(cat $MAESTRO_CERTS/device_cert.pem)
+    SYMPHONY_CLIENT_KEY=$(cat $MAESTRO_CERTS/device_private_key.pem)
+
+    mv /tmp/maestro.config maestro.config
+    sed -i "s|{{DEVICE_ID}}|$DEVICE_ID|g" maestro.config
+    sed -i "s|{{DEVICEDB_SRC}}|$DEVICEDB_SRC|g" maestro.config
+    sed -i "s|{{SYMPHONY_HOST}}|$SYMPHONY_HOST|g" maestro.config
+
+    yq w maestro.config symphony.client_key -- "$SYMPHONY_CLIENT_KEY" > maestro.config.new
+    yq w maestro.config.new symphony.client_cert -- "$SYMPHONY_CLIENT_CRT" > maestro.config
+    rm -f maestro.config.new
```

Comment:
Instead of creating a tmp file, why not just:

`--in-place, -i        Edit files in place (no backup - use caution)`

Reply:
+1 here, using `-i` instead of a temporary file will be a lot more elegant. We missed the existence of this parameter yesterday.